MarsMission/Controllers/LandingViewController.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  LandingViewController.swift
+//  MarsMission
+//
+//  Created by Thanyani on 2020/11/03.
+//
+
+import UIKit
+
+class LandingViewController: UIViewController {
+
+    @IBOutlet weak var dailyWeatherReport: UIButton!
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        navigationItem.title = "Mars Mission"
```

Comment:
Localise this string

Reply:
<img width="532" alt="Screenshot 2020-11-05 at 05 55 52" src="https://user-images.githubusercontent.com/55865505/98195960-a3f92100-1f2b-11eb-988d-dde67cfcc200.png">